components/insight/SRC/org/openmicroscopy/shoola/agents/fsimporter/chooser/FileSelectionTable.java

```diff
@@ -358,7 +358,7 @@ private void removeSelectedFiles()
 	    int[] rows = table.getSelectedRows();
 	    if (rows == null || rows.length == 0) return;
 	    DefaultTableModel dtm = (DefaultTableModel) table.getModel();
-	    Vector<?> v = dtm.getDataVector();
+	    Vector v = dtm.getDataVector();
```

Comment:
just tested with latest build:
``build/src/components/insight/SRC/org/openmicroscopy/shoola/agents/fsimporter/chooser/FileSelectionTable.java:368: error: no suitable method found for setDataVector(Vector<CAP#1>,Vector<String>)
	    dtm.setDataVector(v, this.columnHeadings);
	       ^
    method DefaultTableModel.setDataVector(Vector<? extends Vector>,Vector<?>) is not applicable
      (argument mismatch; Vector<CAP#1> cannot be converted to Vector<? extends Vector>)
    method DefaultTableModel.setDataVector(Object[][],Object[]) is not applicable
      (argument mismatch; Vector<CAP#1> cannot be converted to Object[][])
  where CAP#1 is a fresh type-variable:
    CAP#1 extends Object from capture of ?
``